src/shadowbox/server/main.ts

```diff
@@ -186,9 +186,12 @@ async function main() {
   await accessKeyRepository.start(new RealClock());
 }
 
+function relativePathFromOutlineRoot(file:string): string {
```

Comment:
I found the name confusing, since it's returning an absolute filename.
Maybe `getAbsoluteOutlineFilename()`? It will also better mirror the other method.